project/apps/web/app/profile/page.tsx

```diff
@@ -66,158 +59,46 @@ const ProfilePageContent = () => {
-                <FormItem className="flex flex-row items-center gap-6">
-                  <FormLabel className="text-base min-w-20">Email</FormLabel>
-                  <div className="flex flex-row justify-center gap-3">
-                    <FormControl>
-                      <Input
-                        defaultValue={user?.email}
-                        disabled={!isEditingEmail}
-                        {...field}
-                      />
-                    </FormControl>
-                    {isEditingEmail ? (
-                      <>
-                        <Button
-                          type="button"
-                          disabled={confirmLoading}
-                          onClick={() => handleUpdate('email')}
-                        >
-                          Save
-                        </Button>
-                        <Button
-                          onClick={handleCancelEmail}
-                          variant="outline"
-                          disabled={confirmLoading}
-                        >
-                          Cancel
-                        </Button>
-                      </>
-                    ) : (
-                      <Button
-                        type="button"
-                        disabled={confirmLoading}
-                        onClick={() => setIsEditingEmail(true)}
-                      >
-                        Edit
-                      </Button>
-                    )}
-                  </div>
-                </FormItem>
-              )}
-            />
-            <FormField
-              name="password"
-              render={() => (
-                <FormItem className="flex flex-row items-center gap-6">
-                  <FormLabel className="text-base min-w-20">Password</FormLabel>
-                  <Button
-                    type="button"
-                    className="min-w-[174px]"
-                    disabled={confirmLoading}
-                    onClick={() => setChangePasswordModalOpen(true)}
-                  >
-                    Change Password
-                  </Button>
-                </FormItem>
-              )}
+      <div className="flex flex-col gap-8 p-8 my-8 border border-gray-200 rounded-lg shadow-md">
+        <div className="flex flex-row items-center gap-9">
+          <Avatar className="w-32 h-32">
+            <AvatarImage />
+            <AvatarFallback className="text-xl">
+              {user?.username[0]}
+            </AvatarFallback>
+          </Avatar>
+          <div className="flex flex-col gap-3 items-left">
+            <ProfileDetails
+              user={user}
+              isEditingUsername={isEditingUsername}
+              setIsEditingUsername={setIsEditingUsername}
+              isEditingEmail={isEditingEmail}
+              setIsEditingEmail={setIsEditingEmail}
+              confirmLoading={confirmLoading}
+              onUpdate={handleUpdate}
             />
```

Comment:
since you are using a store, this variables should not need to be passed in as props right?